CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# CTFNote - Contribution Guide
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners/moderators of this repository before making a change.
```

Comment:
What's a moderator?
I think we should use the more generic term "contributors" or "collaborators". The later is what Github uses.